src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -625,26 +604,25 @@ bool LedgerConsensusImp::haveConsensus ()
     // Count number of agreements/disagreements with our position
     for (auto& it : peerPositions_)
     {
-        if (!it.second->isBowOut ())
+        if (!it.second.isBowOut ())
```

Comment:
If you reverse the logic and `continue`, then you can unindent the entire thing one block. I _think_ that looks a lot cleaner; ymmv.